mne_bids/utils.py

```diff
@@ -503,6 +503,28 @@ def _get_brainvision_paths(vhdr_path):
     return (eeg_file_path, vmrk_file_path)
 
 
+def copyfile_ctf(src, dest):
+    """Copy and rename CTF files to a new location.
+
+    Parameters
+    ----------
+    src, dest : str
+        path to the source raw .ds folder and the destination path of the
+        new bids folder.
+    """
+    sh.copytree(src, dest)
+    # list of file types to rename
+    file_types = ('.acq', '.eeg', '.hc', '.hist', '.infods', '.infods.bak',
+                  '.meg4', '.newds', '.res4')
+    # Rename files in dest with the name of the dest directory
+    files_to_rename = [f for f in os.listdir(dest) if f.endswith(file_types)]
+    bids_folder_name = op.splitext(op.split(dest)[-1])[0]
+    for file in files_to_rename:
```

Comment:
`file` is a reserved keyword in python 

you might want to use another variable name

Reply:
Ok thanks